pkg/apis/kubeflow.org/v2alpha1/trainjob_types.go

```diff
@@ -256,6 +264,10 @@ type TrainJobStatus struct {
 	ReplicatedJobsStatus []jobsetv1alpha2.ReplicatedJobStatus `json:"replicatedJobsStatus,omitempty"`
 }
 
+var (
+	TrainJobKind = reflect.TypeOf(TrainJob{}).Name()
```

Comment:
```suggestion
	TrainJobKind string = "TrainJob"
```

I think that this type of reflection does not have value.